docs/Guides/Migration-Guide-V4.md

```diff
@@ -6,6 +6,30 @@ Before migrating to v4, please ensure that you have fixed all deprecation
 warnings from v3. All v3 deprecations have been removed and they will no longer
 work after upgrading.
 
+## Codemods
+### Fastify v4 Codemods
+
+To help with the upgrade, we’ve worked with the team at codemod.com to
+publish codemods that will automatically update your code to many of 
+the new APIs and patterns in Fastify v4.
+Run the following codemods to automatically update your code for Fastify v4 migration:
+
+```
+npx codemod@latest fastify/4/migration-recipe
+```
+
+This will run the following codemods from the Fastify Codemod repository:
+
+- **fastify/4/remove-app-use**
+- **fastify/4/reply-raw-access**
+- **fastify/4/wrap-routes-plugin**
+- **fastify/4/await-register-calls**
+
+Each of these codemods automates the changes listed in the v4 migration guide.
+For a complete list of available Fastify codemods and further details, 
+see the [codemod registry](https://codemod.com/registry?q=fastify).
+
+
```

Comment:
You can add:

If you have any feedback for codemods run this command:
npx codemod feedback